docs/source/ecf.rst

```diff
@@ -35,6 +35,103 @@ If True, skip the named step.
 .. note::
    Note that some instruments and observing modes might skip a step either way! See the `calwebb_detector1 docs <https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_detector1.html>`__ for the list of steps run for each instrument/mode by the STScI's JWST pipeline.
 
+custom_bias
+'''''''''''''''''
+Boolean, allows user to supply a custom bias file and overwrite the default file
+
+superbias_file
+'''''''''''''''''
+path to custom superbias file
+
+update_sat_flags
+'''''''''''''''''
+Boolean, allows user to have more control over saturation flags
```

Comment:
Could you add more details here.  My impression is that this must be set to `True` in order to use some of the commands below (e.g., dq_sat_mode), but I'm not certain on that.